persistence-jdbc/scaladsl/src/main/resources/reference.conf

```diff
@@ -0,0 +1,82 @@
+play.modules.enabled += com.lightbend.lagom.scaladsl.persistence.jdbc.JdbcPersistenceModule
+
+//#persistence
+# Defaults to use for each Akka persistence plugin
+jdbc-defaults.slick {
+
+  # The driver to use
+  driver = "slick.driver.H2Driver$"
+
+  # The JNDI name
+  jndiName = "DefaultDS"
+}
+
+# Configure the default database to be bound to JNDI
+db.default.jndiName=DefaultDS
+
+lagom.persistence.jdbc {
```

Comment:
shall we move all of these settings to `lagom-persistence-jdbc-core`?
feels dangerous to have same keys in multiple jars, even though they should not be used at the same time